examples/cts1-multi/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "cts1-multi"
+version = "0.1.0"
+authors = ["Concordium <developers@concordium.com>"]
+edition = "2018"
+license = "MPL-2.0"
+
+[features]
+default = ["std"]
+std = ["concordium-std/std", "concordium-cts1/std"]
+
+[dependencies]
+concordium-std = {path = "../../concordium-std", default-features = false}
+concordium-cts1 = {path = "../../concordium-cts1", default-features = false}
+
+[lib]
+crate-type=["cdylib", "rlib"]
+
+[profile.release]
+opt-level = "z"
```

Comment:
Have you tried `codegen-units=1` as we discussed? Does it improve the size?

Reply:
I have added codegen-units since it reduces the size